Phat-Taupe-Pests/BangazonAPI
# Eliza template

***********

DESCRIPTION
	Put together the file structure, including context, Startup.cs. Wrote Customer Model and Controller, migrated then, and got them to put/post/getAll/getSingle.

STEPS TO TEST
        0. Set up environment variable. Mine looks like this, edit for your system (use pwd to find the path in your directory): 

export BANGAZON_DB="/path/to/repo/BangazonAPI/bangazonapi.db"

        0.5. Database file should be named bangazonapi.db
	1. Clone this branch to a local repo
	2. In terminal, run ```dotnet build``` then ```dotnet run```
	3. Test functionality
		A. Use POSTMAN to post, put, getSingle and getAll customers.
		B. POST url is http://localhost:5000/Customer
		C. PUT url is http://localhost:5000/Customer/{id}
                D. GET single url is http://localhost:5000/Customer/{id}
                E. GET all url is http://localhost:5000/Customer

REFERENCE
	[Ticket](https://github.com/Phat-Taupe-Pests/BangazonAPI/issues/1)
	[ERD](https://github.com/Phat-Taupe-Pests/Bangazon)
	
***********
